src/lsp/superbol_free_lib/vscode_extension.ml

```diff
@@ -546,3 +558,6 @@ let manifest =
       "workspace";                                 (* <- run on the workspace *)
     ]
     ~contributes
+    ~extensionDependencies: [
+      "JacquesLucke.gcov-viewer";
```

Comment:
Is this is hard dependency or an optional one ?

Reply:
Hard, I can make it optional, but I don't know how well the execution of the commands would work if the extension is missing

Reply:
I'd go with a hard dependency as availability seems universal, at least until we have a case where this is an issue.